### PR TITLE
task15

### PR DESCRIPTION
**Pull request by vramaniuk**

### CODE REVIEW

File: 1-select-tasks/22.txt [hunk 1]
```diff
@@ -1,3 +1,17 @@
 Create a SQL query to display the name of each customer along with their most expensive purchased product:
 
-| CompanyName | ProductName | PricePerItem |
```
Comment (AlexMokin) [inline 1-select-tasks/22.txt]: One task per pull request!

File: 1-select-tasks/15.txt [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+   [6] AS 'June',
+   [7] AS 'July',
+   [8] AS 'August',
+   [9] AS 'September',
+   [10] AS 'October',
+   [11] AS 'November',
+   [12] AS 'December' FROM(
+SELECT MONTH([o].[OrderDate]) AS [Months], COUNT(*) AS 'OrdersSum'
+FROM [Orders] [o]
+WHERE [o].[OrderDate] LIKE '%1997%'
```
Comment (AlexMokin) [inline 1-select-tasks/15.txt]: use [year](https://docs.microsoft.com/en-us/sql/t-sql/functions/year-transact-sql) function

**Comment by vramaniuk** [comment]
Sorry, I changed another file in a branch task15 accidentally
 and actually didn`t want to commit many solutions in one time. I fixed it. See af362f9 commit

**Comment by AlexMokin** [comment]
well done!

**Pull request closed without merging**